internal/manifest/version.go

```diff
@@ -865,14 +861,14 @@ func (m *FileMetadata) Validate(cmp Compare, formatKey base.FormatKey) error {
 		return base.CorruptionErrorf("file metadata FileBacking not set")
 	}
 
-	if m.PrefixReplacement != nil {
+	if m.SyntheticPrefix.IsSet() {
 		if !m.Virtual {
 			return base.CorruptionErrorf("prefix replacement rule set with non-virtual file")
```

Comment:
uber nit: these error messages could be updated to use the phrase "synthetic prefix" instead of "prefix replacement"